lib/office.rb

```diff
@@ -0,0 +1,8 @@
+class Office
+
+  def create_offices(data)
+    data.map do |office_data|
+      Facility.new(office_data)  
```

Comment:
Because you do all the parsing in the Facility class, this is a really clean method! 

Another option could be to have those parsers here. It might make more sense to call this class `Office_Factory` or something to indicate that its purpose to create facilities (it's not really a representation of an Office)